modules/utils/src/main/java/io/uhndata/cards/serialize/internal/DefaultOptionsProcessor.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package io.uhndata.cards.serialize.internal;
+
+import java.util.function.Function;
+
+import javax.jcr.Node;
+import javax.jcr.RepositoryException;
+import javax.json.JsonValue;
+
+import org.osgi.service.component.annotations.Component;
+
+import io.uhndata.cards.serialize.spi.ResourceJsonProcessor;
+
+/**
+ * Load the {@code defaultOptions} child node. The name of this processor is {@code identify}.
+ *
+ * @version $Id$
+ */
+@Component(immediate = true)
+public class DefaultOptionsProcessor implements ResourceJsonProcessor
+{
+    @Override
+    public String getName()
+    {
+        return "includeDefaultOptions";
+    }
+
+    @Override
+    public int getPriority()
+    {
+        return 10;
+    }
+
+    @Override
+    public JsonValue processChild(final Node node, final Node child, final JsonValue input,
+        final Function<Node, JsonValue> serializeNode)
+    {
+        try {
+            if (child.getName().equals("defaultOptions")) {
```

Comment:
Even though `child` is expected to have a `name`, flipping the two is always safer:
```suggestion
            if ("defaultOptions".equals(child.getName())) {
```